mkdocs.yml

```
@@ -43,3 +43,7 @@ nav:
 plugins:
     - search
     - mkdocstrings
+
+extra:
+    version:
+        provider: mike
```

A carriage return is missing here
```suggestion
        provider: mike

```